### ISSUE TITLE
Bug: `Cannot read properties of undefined (reading 'url')`

### ISSUE THREAD

**Issue by EvHaus** [AtomLinter/linter-eslint-node#18]
## Issue Type

Bug

## Issue Description

When using `linter-eslint-node` I frequently get this:

<img width="475" alt="Screen Shot 2022-03-25 at 9 14 58 AM" src="https://user-images.githubusercontent.com/226640/160159522-1ac08b18-3b5a-487d-8ff1-2f2f41eca4fa.png">

```
Cannot read properties of undefined (reading 'url')
```

There is no tracestack in the dev tools so I'm not sure what's causing it. The issue seems to go away on its own after saving the file, but then comes back again at some point. I'm still not able to figure out how to reproduce consistently.

## Bug Checklist
- [x] Restart Atom
- [x] Verify the `eslint` CLI gives the proper result, while `linter-eslint-node` does not
- [x] Paste the output of the `Linter Eslint Node: Debug` command from the Command Palette below

```
Atom version: 1.61.0-beta0
linter-eslint-node version: 1.0.4
Worker using Node at path: /Users/evhaus/.nvm/versions/node/v16.14.1/bin/node
Worker Node version: v16.14.1
Worker PID: 7240
ESLint version: 8.11.0
ESLint location: /Users/evhaus/Git/zenhub/node_modules/eslint/
Linting in this project performed by: linter-eslint-node
Hours since last Atom restart: 0.5
Platform: darwin
Current file's scopes: [
  "source.ts",
  "punctuation.terminator.statement.semicolon"
]
linter-eslint-node configuration: {
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.flow",
    "source.babel",
    "source.js-semantic",
    "source.ts"
  ],
  "nodeBin": "node",
  "warnAboutOldEslint": true,
  "autofix": {
    "fixOnSave": false,
    "rulesToDisableWhileFixing": [],
    "ignoreFixableRulesWhileTyping": false
  },
  "disabling": {
    "disableWhenNoEslintConfig": true,
    "rulesToSilenceWhileTyping": []
  },
  "advanced": {
    "disableEslintIgnore": false,
    "showRuleIdInMessage": true,
    "useCache": true
  }
}
```

**Comment by savetheclocktower** [comment]
I think #17 will fix this. I'll try to get it out today.

**Comment by savetheclocktower** [comment]
OK, closing based on my assumption that this is fixed in 1.0.5, but if you update and it's still happening, let me know and I can reopen. Thanks!

**Comment by EvHaus** [comment]
Thanks for the super fast response and fix. Testing 1.0.5 now and will report back if it's still an issue.
